Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/ChangePlayerStatusMessage.java

```diff
@@ -0,0 +1,34 @@
+package nl.soccar.gameserver.controller.socnet.message;
+
+import nl.soccar.gameserver.controller.socnet.message.handler.ChangePlayerStatusMessageHandler;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+
+/**
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.CHANGE_PLAYER_STATUS_MESSAGE_ID, handler = ChangePlayerStatusMessageHandler.class)
+public final class ChangePlayerStatusMessage extends Message {
+
+    private final Status status;
+
+    public ChangePlayerStatusMessage(Status status) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)